FreewheelSampleApp/app/build.gradle

```diff
@@ -30,9 +30,18 @@ task copyTask(type: Copy) {
 
 tasks.copyTask.execute()
 
+project.configurations.all {
+    resolutionStrategy.eachDependency { details ->
+        if (details.requested.group == 'com.android.support'
+                && !details.requested.name.contains('multidex')) {
+            details.useVersion "27.1.1"
```

Comment:
version should be extracted as constant. If we update appcompat version this code might downgrade version for other support libraries